manifest.json

```diff
@@ -247,9 +247,11 @@
                 "CreateServiceSpecificCredential",
                 "CreateUser",
                 "DeleteAccessKey",
+                "DeleteServiceSpecificCredential",
```

Comment:
Don't forget the changelog ;)

Reply:
Thanks for the reminder @jderusse - updated :)